src/components/utils/onfidoApi.js

```diff
@@ -46,8 +46,28 @@ export const sendMultiframeSelfie = (snapshot, selfie, token, url, onSuccess, on
     const snapshot_uuids = JSON.stringify([res.uuid])
     uploadLivePhoto({ file: { blob, filename }, sdkMetadata, snapshot_uuids}, url, token, onSuccess, onError)
   })
-  .catch((res) => {
-    onError(res)
+  .catch(() => {
+    // TODO when the backened enpoint will be ready please uncomment this line
+    // onError(res)
+
+    // TODO when the backened enpoint will be ready please delete this code
```

Comment:
Minor: typos (lines 50, 53, 63) - 1 too many "e" in "backend" & missing "d" in "endpoint" ;)